### PR TITLE
Update training availability

### PR DESCRIPTION
**Pull request by bonnieAcameron**
[Preview --->](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/training/)

### CODE REVIEW

File: _pages/about/index.md
Comment (mejiaj) [inline _pages/about/index.md:1]: [About →](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/about/)

File: _pages/training/create-your-own-class.md
Comment (mejiaj) [inline _pages/training/create-your-own-class.md:1]: [Create your own class →](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/training/create-your-own-class/)

File: _pages/training/index.md
Comment (mejiaj) [inline _pages/training/index.md:1]: [Training →](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/training/)

File: _pages/training/online-training.md
Comment (mejiaj) [inline _pages/training/online-training.md:1]: [Online training →](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/training/online-training/)

File: _pages/training/tips-for-trainers.md
Comment (mejiaj) [inline _pages/training/tips-for-trainers.md:1]: [Tips for starting a plain language program →](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/training/tips-for-trainers/)

File: _pages/training/videos.md
Comment (mejiaj) [inline _pages/training/videos.md:1]: [Videos →](https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/training/videos/)

File: _pages/training/videos.md [hunk 1]
```diff
@@ -10,39 +10,30 @@ redirect_from:
...
-- [2018 PLAIN Summit](https://www.youtube.com/embed/W4gzfemW0Rc)
 - [PLAIN](https://www.youtube.com/playlist?list=PLd9b-GuOJ3nHMlmPFMw8cJxN_DW-odj0J)
 - [U.S. Citizenship and Immigration Services](https://www.youtube.com/playlist?list=PLADE80C67FDB39352)
 - [Purdue Online Writing Lab (OWL)](https://www.youtube.com/user/OWLPurdue/playlists)
 
 ## PLAIN Meetings
 
--  [June 2021 Meeting: Using Words to Build Better Digital Experiences](https://digital.gov/event/2021/06/09/plain-language-webinar-using-words-to-build-better-digital-experiences/)
+- [2019 PLAIN Summit](https://www.youtube.com/embed/cny1CIMuv3o)
+- [2018 PLAIN Summit](https://www.youtube.com/embed/W4gzfemW0Rc)
```
Comment (mejiaj) [inline _pages/training/videos.md:22]: Should this be a playlist instead of an embed? The video shows blown up.

**Review by mejiaj — COMMENTED** [review]
LGTM, added preview links to each resource. 

Only a question on the `2019 PLAIN Summit` link in https://federalist-dad55c1b-07c6-4c7e-89da-5aff1f9335fd.sites.pages.cloud.gov/preview/gsa/plainlanguage.gov/bc-update-training/training/videos/.

File: _pages/training/videos.md (hunk 1 above)
Comment (bonnieAcameron) [reply _pages/training/videos.md:22]: Yes, thank you. Will update.

**Comment by nick-mon1** [comment]
@bonnieAcameron your updates look good. No visual regression issues were found.

**Review by mejiaj — APPROVED** [review]

**Pull request merged**